pkg/helmdeployer/deployer.go

```diff
@@ -38,7 +41,21 @@ const (
 	DefaultServiceAccount        = "fleet-default"
 )
 
-var ErrNoRelease = errors.New("failed to find release")
+var (
+	ErrNoRelease       = errors.New("failed to find release")
+	DefaultKey         = "values.yaml"
+	SecretSeparator    = ": "
+	ConfigMapSeparator = "="
```

Comment:
As a user, if I name a file `values.yaml` I will make the content YAML format, so it is `key: value`, not `key=value`  no matter I save the file in a Secret or a ConfigMap.

@nickgerace @kinarashah  What do you think?

Reply:
Discussed offline. Format to be validated based on examples from fluxcd - [secret](https://github.com/fluxcd/helm-controller/pull/44/files#diff-0c22d9a7085f353b1af8ac204842b0f903d6d61480ee5838048c4cf7b0994f31), [configmap](  https://github.com/fluxcd/helm-controller/pull/44/files#diff-3a3a972d6b8cae7337d4a238d6ef3f79adc9c0d15df6f978f086336da85b13f6) and [spec](https://github.com/fluxcd/helm-controller/pull/44/files#diff-0729f6d93c33d655766cf9c951f6ed913c1cc7eaed4601a957de7de55fd5779d).